.storybook/stories/reactstrap-installation.stories.mdx

```diff
@@ -0,0 +1,104 @@
+<!-- reactstrap-installation.stories.mdx -->
+
+import { Meta } from '@storybook/addon-docs';
+import { Button } from 'reactstrap';
+
+<Meta title="3rd Party/Reactstrap/Installation" />
+
+### Reactstrap is a [React](https://reactjs.org) component library for [Bootstrap](https://getbootstrap.com)
+
+The Availity UI Kit utilizes Bootstrap 4, which caps the Reactstrap version to [Reactstrap v8](https://deploy-preview-2356--reactstrap.netlify.app/)
+
+## Getting Started
+
+### Install Reactstrap:
+
+```sh
+npm install reactstrap react react-dom
```

Comment:
I think we should specify a version for the `reactstrap` package. eg: `reactstrap@^8.0.0`